dymos/phase/phase.py

```diff
@@ -524,11 +514,9 @@ def check_parameter(self, name):
             raise ValueError(f'{name} has already been added as a control.')
         elif name in self.parameter_options:
             raise ValueError(f'{name} has already been added as a parameter.')
-        elif name in self.polynomial_control_options:
-            raise ValueError(f'{name} has already been added as a polynomial control.')
 
-    def add_control(self, name, units=_unspecified, desc=_unspecified, opt=_unspecified,
-                    fix_initial=_unspecified, fix_final=_unspecified, targets=_unspecified,
+    def add_control(self, name, control_type=_unspecified, order=_unspecified, units=_unspecified, desc=_unspecified,
```

Comment:
We should put control_type and order at the end of the arguments list here.  Anyone using positional arguments would have their code broken by this change.

We should also raise deprecation warnings for `add_polynomial_control`

Reply:
Good call, I've reordered `control_type` to be at the end everywhere.

I have a deprecation warning for `add_polynomial_control` like this, let me know if you want to see a change:

```
        om.issue_warning(f'{self.pathname}: The method `add_polynomial_control` is '
                         'deprecated and will be removed in Dymos 2.1. Please use '
                         '`add_control` with the appropriate options to define a polynomial control.',
                         category=om.OMDeprecationWarning)
```